### PR TITLE
VIH-6732 additional logging

### PR DESCRIPTION
**Pull request by levibutler**
Vih-6732 

### Change description ###
Additional logging

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X ] No
```

### CODE REVIEW

File: VideoAPI/Video.API/Controllers/AudioRecordingController.cs [hunk 1]
```diff
@@ -173,7 +173,7 @@ public async Task<IActionResult> GetAudioStreamMonitoringInfoAsync(Guid hearingI
         [ProducesResponseType((int)HttpStatusCode.NotFound)]
         public async Task<IActionResult> GetAudioRecordingLinkAsync(Guid hearingId)
         {
-            _logger.LogInformation($"Getting audio recording link for hearing: {hearingId}");
+            _logger.LogInformation("Getting audio recording link");
             try
```
Comment (shaed-parkar) [inline VideoAPI/Video.API/Controllers/AudioRecordingController.cs:177]: Did you mean to remove hearing id from the log altogether (i.e. not even as an argument)?

Comment (levibutler) [reply VideoAPI/Video.API/Controllers/AudioRecordingController.cs:177]: theyre added to the scope elsewhere so will still appear in properties, i thought by removing we could easier group logs easier

File: VideoAPI/Video.API/Startup.cs [hunk 2]
```diff
@@ -127,14 +129,14 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
             }
 
             app.UseRouting();
-            
+
+            //app.UseMiddleware<LoggingMiddleware>();
```
Comment (shaed-parkar) [inline VideoAPI/Video.API/Startup.cs:133]: Would you mind deleting this if it's not required anymore?

File: VideoAPI/Video.API/Controllers/AudioRecordingController.cs (hunk 1 above)
Comment (shaed-parkar) [reply VideoAPI/Video.API/Controllers/AudioRecordingController.cs:177]: Never mind, I saw the implementation of the data extractor

**Review by shaed-parkar — APPROVED** [review]
LGTM. Can't wait to see this in action!

File: VideoAPI/Video.API/Startup.cs (hunk 2 above)
Comment (levibutler) [reply VideoAPI/Video.API/Startup.cs:133]: done

**Review by shaed-parkar — APPROVED** [review]
LGTM

**Pull request merged**